### PR TITLE
fix #78792 pinnacle.com

### LINKED ISSUE #78792: www.pinnacle.com
### Issue URL (Annoyance)
 [https://www.pinnacle.com/de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.pinnacle.com%2Fde%2F)
### Comment
Username: @Morku
> Cookie Banner

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by wout-codes**
#78792

### CODE REVIEW

File: AnnoyancesFilter/sections/self-promo.txt [hunk 1]
```diff
@@ -52,6 +52,8 @@ disk.yandex.*##.mail360-banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/78285
 nu.nl##.zone > div[id^="block-"][data-context*="Adverteerders"]
 nu.nl##.zone > div[id^="block-"][data-context*="Aangeboden door NUshop"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/78792
+pinnacle.com##a[data-gtm-id^="promo_banner_"][rel^="noopener"]
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:56]: Do you need `[rel^="noopener"]`? `pinnacle.com##a[data-gtm-id^="promo_banner_"]` works ok on my end.

Comment (wout-codes) [reply AnnoyancesFilter/sections/self-promo.txt:56]: Yeah, it works too. Just to narrow it down.

Comment (zloyden) [reply AnnoyancesFilter/sections/self-promo.txt:56]: According to the code, without `[rel^="noopener"]` should be ok.

Comment (wout-codes) [reply AnnoyancesFilter/sections/self-promo.txt:56]: Okay. I will remove it.

Comment (wout-codes) [reply AnnoyancesFilter/sections/self-promo.txt:56]: Okay. I will remove it.

**Review by zloyden — APPROVED** [review]

**Pull request merged**